package-res/resources/web/dataapi/models-svc.js

```diff
@@ -55,7 +55,7 @@ pentaho.pda.SvcHandler.prototype.getSources = function(callback, options) {
             var contextName = options['context'];
             var query = "domainName=&"+((contextName)? 'context='+contextName : "");
             var jsonStr = pentahoPost( url, query, null, 'text/text' );
```

Comment:
no, there are probably tons of uses of it out there still and we'd have to make it return exactly what it returns now, which is potentially still using eval, so it wouldn't really provide the whole solution without refactoring all of the places that use it as well.